.github/workflows/ci.yml

```diff
@@ -72,3 +72,53 @@ jobs:
             cd build && cmake -DBUILD_BPF=On -DUSE_BUNDLED_DEPS=False ../
             KERNELDIR=/lib/modules/$(ls /lib/modules)/build make -j4
             make run-unit-tests
+
+  build-and-test-modern-bpf-x86:
+    name: build-and-test-modern-bpf-x86 🏗️🧪
+    runs-on: ubuntu-22.04
+    steps:
+
+      - name: Checkout Libs ⤵️
+        uses: actions/checkout@v3
+
+      - name: Install deps ⛓️
+        run: |
+          sudo apt update 
+          sudo apt install -y --no-install-recommends ca-certificates cmake build-essential clang-14 git pkg-config autoconf automake libtool libelf-dev
+
+      - name: Build scap-open and check no verifier issues 👀
+        run: |
+          mkdir -p build
+          cd build && cmake -DUSE_BUNDLED_DEPS=ON -DUSE_MODERN_BPF=ON -DBUILD_MODERN_BPF_TEST=ON -DMODERN_BPF_DEBUG_MODE=ON -DBUILD_LIBSCAP_GVISOR=OFF ../
+          make scap-open
+          sudo ./libscap/examples/01-open/scap-open --modern_bpf --num_events 0
```

Comment:
I know I'm being super annoying right now, but wouldn't running the test already check any verifier errors? Maybe we can build `scap-open` and go straight into running the tests. Unless the tests don't output the verifier message to the console in which case the discussion would be a different one

Reply:
Uhm that's a good point! Yes theoretically the tests should already check that there are no verifier issues, but right now running the `scap-open` is the only way we have to check that there are no regressions in the communication between the probe and the userspace like here https://github.com/falcosecurity/libs/pull/483... So probably I would keep the running phase just to be sure, but let me know if it is not good for you, or if you see any disadvantages in doing this 

Maybe i will split again the steps as said here https://github.com/falcosecurity/libs/pull/496#discussion_r927509917 :point_down:

Reply:
I can also change the job name from ` check no verifier issues 👀`  to `run scap-open` to be clearer

Reply:
I think it's fine, as long as the 2 commands serve separate purposes, I wasn't aware of the probe/userspace communication problems, we can leave this as is.